src/py42/clients/settings/device_settings.py

```diff
@@ -0,0 +1,349 @@
+    for `DeviceSettings` to manage individual device settings."""
+
+    def __init__(self, device_dict, org_settings):
+        self.data = device_dict
+        self._org_settings = org_settings
+        self.changes = org_settings.changes
+        self._destinations = org_settings.data["settings"]["destinations"]
+        self.data["settings"] = {
+            "serviceBackupConfig": self.data["serviceBackupConfig"]
+        }
+        self.backup_sets = [
+            BackupSet(self, self.changes, backup_set)
+            for backup_set in self.data["serviceBackupConfig"]["backupConfig"][
+                "backupSets"
+            ]
+        ]
+
+    @property
+    def available_destinations(self):
+        return {d["guid"]: d["destinationName"] for d in self._destinations}
+
+    warning_email_enabled = SettingProperty(
+        name="warning_email_enabled",
+        location=["settings", "serviceBackupConfig", "warningEmailEnabled"],
+        get_converter=str_to_bool,
+        set_converter=bool_to_str,
+    )
+    critical_email_enabled = SettingProperty(
+        name="critical_email_enabled",
+        location=["settings", "serviceBackupConfig", "severeEmailEnabled"],
+        get_converter=str_to_bool,
+        set_converter=bool_to_str,
+    )
+    warning_alert_days = SettingProperty(
+        name="warning_alert_days",
+        location=["settings", "serviceBackupConfig", "minutesUntilWarning"],
+        get_converter=minutes_to_days,
+        set_converter=days_to_minutes,
+    )
+    critical_alert_days = SettingProperty(
+        name="critical_alert_days",
+        location=["settings", "serviceBackupConfig", "minutesUntilSevere"],
+        get_converter=minutes_to_days,
+        set_converter=days_to_minutes,
+    )
+    backup_status_email_enabled = SettingProperty(
+        name="backup_status_email_enabled",
+        location=["settings", "serviceBackupConfig", "backupStatusEmailEnabled"],
+        get_converter=str_to_bool,
+        set_converter=bool_to_str,
+    )
+    backup_status_email_frequency_days = SettingProperty(
+        name="backup_status_email_frequency_days",
+        location=["settings", "serviceBackupConfig", "backupStatusEmailFreqInMinutes"],
+        get_converter=minutes_to_days,
+        set_converter=days_to_minutes,
+    )
+
+    def __repr__(self):
+        return "<DeviceSettingsDefaults: org_id: {}>".format(self._org_settings.org_id)
+
+
+class DeviceSettings(DeviceSettingsDefaults):
+    """Class used to manage an individual device's settings."""
+
+    def __init__(self, device_dict):
+        self.changes = {}
+        self.data = device_dict
+        self._destinations = device_dict["availableDestinations"]
+        self.backup_sets = [
+            BackupSet(self, self.changes, backup_set)
+            for backup_set in self.data["settings"]["serviceBackupConfig"][
+                "backupConfig"
+            ]["backupSets"]
+        ]
+
+    @property
+    def computer_id(self):
```

Comment:
This should probably be `device_id`. We call `computer_id` `device_id` in other places in py42.

Reply:
@timabrmsn thoughts here?

Reply:
I'm a bit reluctant, since `computerId` is the actual key in the underlying setting dict, and since that's the technical term used in our API docs and in code. But you're right, if someone's knowledge of our stuff is only from py42 they'd be inclined to look for `device_id`... 

I suppose we could just expose another property named `device_id` so you can find it regardless of what term you're looking for. Just label one as an alias of the other?